### ISSUE TITLE
Webtoons

### ISSUE THREAD

**Issue by C9Glax** [metafates/mangal#153]
### Source URL

https://www.webtoons.com/

### Checks

- [X] This source is SFW

### Additional Context

To access some comics, you have to authenticate with an account.

**Comment by amitSA** [comment]
Is Webtoon supported ? I don't see it in the custom scrapers directory

**Comment by C9Glax** [comment]
No, and since this repo hasn't received updates in a year, it is pretty dead.
